posting/mvcc.go

```diff
@@ -86,6 +87,53 @@ func (ir *incrRollupi) rollUpKey(writer *TxnWriter, key []byte) error {
 	return writer.Write(&bpb.KVList{Kv: kvs})
 }
 
+// rollupDeltas will try to collect 1000 deltas for roll up within 30 seconds.
+func (ir *incrRollupi) rollupDeltas() {
+	var wg sync.WaitGroup
+	wg.Add(1000) // Push 1,000 deltas for rollup.
+	ctx, cancel := context.WithTimeout(context.Background(), 30*time.Second)
+
+	stream := pstore.NewStreamAt(math.MaxUint64)
+	stream.Prefix = []byte{x.DefaultPrefix}
+	stream.Logger = nil // We don't want any logs. This can get very noisy.
+	stream.ChooseKey = func(item *badger.Item) bool {
+		pk, err := x.Parse(item.Key())
+		if err != nil {
+			glog.Errorf("RollupDeltas Parse failed: err = %s", err)
+			return false
+		}
+		if pk.IsData() && item.UserMeta()&BitDeltaPosting > 0 {
```

Comment:
Maybe we should try to randomize this. I think it might be possible that some keys would never get rolled up. @manishrjain is there a way to iterate over all data keys?